step-templates/sqlserver-high-availability-group.json

```diff
@@ -0,0 +1,48 @@
+    "Octopus.Action.Script.ScriptFileName": null,
+    "Octopus.Action.Package.FeedId": null,
+    "Octopus.Action.Package.PackageId": null
+  },
+  "Parameters": [
+    {
+      "Id": "afe518f5-5778-45b4-86c7-3715fa55b4d9",
+      "Name": "SQLServer",
```

Comment:
@SDFVasantHorapeti Thanks for the submission. I think the "prefix your parameter names" instruction is a bit unclear. We mean these parameters, not the variables in the script. The reason is clashes with the project variables.

I suggest `HAGroupSQLServer` and `HAGroupPrimaryNode`

Reply:
@droyad Thanks for your review comments. I have renamed the parameters and committed the changes.